src/switch_rtp.c

```diff
@@ -8251,7 +8251,8 @@ static int rtp_common_write(switch_rtp_t *rtp_session,
 		}
 
 		/* If the marker was set, and the timestamp seems to have started over - set a new SSRC, to indicate this is a new stream */
-		if (m && !switch_rtp_test_flag(rtp_session, SWITCH_RTP_FLAG_SECURE_SEND) && (rtp_session->rtp_bugs & RTP_BUG_CHANGE_SSRC_ON_MARKER) &&
```

Comment:
Are there any restrictions on changing SSRC in SRTP that I am unaware of?

Reply:
there are legitimate reasons to not do so.

Reply:
@jakubkarolczyk can you detail your statement
"there are legitimate reasons to not do so."
Are you saying we can not change the SSRC with SRTP ?

Reply:
Looking at your comment 
"From your last comment to the connected issue I understood that SSRC change is enough to resolve the problem you observe. So why would you like to instert m bit?"

I guess you are still fine with that, but you just wanted to highlight that there may be issues.

I may dig around that fix and see is this is solving my issues, I just wanted to know why this ended up not being merged/reviewed.